training/train.py

```diff
@@ -302,30 +302,48 @@ def create_git_pr(s3_client, model_version, recall_at_30):  # pragma: no cover
     prev_hyperparams = s3_client.read_json_file(k)
 
     # Convert the json description to string
-    description = json.dumps(prev_hyperparams).replace('"', '\\"')
+    description = 'Previous hyper parameters :: ' + json.dumps(prev_hyperparams).replace('"', '\\"')
 
     prev_recall = prev_hyperparams.get('recall_at_30', 0.55)
+    _logger.info('create_git_pr:: Prev => Model {}, Recall {}  Curr => Model {}, Recall {}'.format(
+        previous_version, prev_recall, model_version, recall_at_30))
     if recall_at_30 >= prev_recall:
         try:
             # Invoke bash script to create a saas-analytics PR
-            t = subprocess.Popen(['sh', 'rudra/utils/github_helper.sh', 'f8a-hpf-insights.yaml',
-                                  'MODEL_VERSION', str(model_version), description],
-                                 shell=False, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-            # Wait for the subprocess to get over
-            t.wait(60)
-            if t.returncode == 0:
-                _logger.info("Successfully created a PR")
+            t1 = subprocess.Popen(['wget', '-v',
+                                   'https://raw.githubusercontent.com/fabric8-analytics/'
+                                   'fabric8-analytics-rudra/master/rudra/utils/github_helper.sh',
+                                   '-O', './github_helper.sh'], shell=False)
+            t1.wait(60)
+            _logger.info('Error while getting github helper, error code: {}'.format(t1.returncode))
```

Comment:
Are you always printing an error log?

Reply:
I am printing logs every time, but I can re-phrase it a bit. 
Yes, this logs are use full as there are very few logs in the entire pipeline flow, it good to know  which sub-process failed. I feel we can be generous in adding logs in re-training flow, because (1) They are all saved in s3 in gz format (2) There are very few logs (less than 200 lines) (3) Flow triggers once in a week, so if we want to investigate past failure this extra logs can help us.